hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCpOptions.java

```diff
@@ -684,11 +684,23 @@ public Builder withAppend(boolean newAppend) {
       return this;
     }
 
+    /**
+     * whether builder with crc.
+     * @param newSkipCRC whether to skip crc check
+     * @return  Builder object whether to skip crc check
+     * @deprecated Use {@link #withSkipCRC(boolean)} instead.
+     */
+    @Deprecated
     public Builder withCRC(boolean newSkipCRC) {
       this.skipCRC = newSkipCRC;
       return this;
     }
 
+    public Builder withSkipCRC(boolean newSkipCRC) {
```

Comment:
copy the javadocs from above, now you've written them

Reply:
> copy the javadocs from above, now you've written them

Thanks @steveloughran  for your notice ,have just updated